spec/amazon_ses_mailer/base_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+RSpec.describe AmazonSesMailer::Base do
```

Comment:
Tests missing for these methods:
- process_merge_vars
- convert_instance_variables_to_merge_vars
- transform_hash
- transform_value

Reply:
Not done yet.